src/modules/AuthService.ts

```diff
@@ -23,7 +23,7 @@ const firebaseConfig = {
 };
 
 const firebaseApp = firebase.initializeApp(firebaseConfig);
-const auth = firebaseApp.auth();
+export const auth = firebaseApp.auth();
```

Comment:
We don't want to expose the `auth` instance, please create appropriate wrapper methods.

Reply:
This is required for auth listener function wrapping this is not working it gives an unknown error. And I cannot find any working fix for this.
```js
TypeError: Cannot read property 'ba' of undefined
    at push../node_modules/@firebase/auth/dist/auth.esm.js.k.Hc (auth.esm.js:394)
    at d (auth.esm.js:420)
    at GlobalContext.tsx:49
    at invokePassiveEffectCreate (react-dom.development.js:23487)
    at HTMLUnknownElement.callCallback (react-dom.development.js:3945)
    at Object.invokeGuardedCallbackDev (react-dom.development.js:3994)
    at invokeGuardedCallback (react-dom.development.js:4056)
    at flushPassiveEffectsImpl (react-dom.development.js:23574)
    at unstable_runWithPriority (scheduler.development.js:468)
    at runWithPriority$1 (react-dom.development.js:11276)
    at flushPassiveEffects (react-dom.development.js:23447)
    at react-dom.development.js:23324
    at workLoop (scheduler.development.js:417)
    at flushWork (scheduler.development.js:390)
    at MessagePort.performWorkUntilDeadline (scheduler.development.js:157)
```

Reply:
Wrapper added